src/Sentry/TransactionTracer.cs

```diff
@@ -215,10 +215,16 @@ public TransactionTracer(IHub hub, string name, string operation, TransactionNam
     /// <summary>
     /// Initializes an instance of <see cref="TransactionTracer"/>.
     /// </summary>
-    public TransactionTracer(IHub hub, ITransactionContext context)
+    public TransactionTracer(IHub hub, ITransactionContext context) : this(hub, context, null)
+    {
+    }
+
+    /// <summary>
+    /// Initializes an instance of <see cref="TransactionTracer"/>.
+    /// </summary>
+    internal TransactionTracer(IHub hub, ITransactionContext context, TimeSpan? idleTimeoutOverride)
```

Comment:
```suggestion
    internal TransactionTracer(IHub hub, ITransactionContext context, TimeSpan? idleTimeout = null)
```
By default, Transactions do not time out. But with this we have the option of providing a timeout.